runtime/doc/treesitter.txt

```diff
@@ -187,6 +187,11 @@ Note: The predicates listed in the web page above differ from those Neovim
 supports. See |lua-treesitter-predicates| for a complete list of predicates
 supported by Neovim.
 
+By default, the first query on `runtimepath` is used (which usually implies
+that user config takes precedence over plugins, which take precedence over
+queries bundled with Neovim). If a query should extend other queries instead
+of replacing them, use the `; extends` modeline below.
+
```

Comment:
Note for the future: we may need a reference to the modeline documentation here.

Reply:
Cleaning up the docs is a separate item on the tracker; let's do that then.